### ISSUE TITLE
Refreshing Data / Updating Line Chart Dynamically 

### ISSUE THREAD

**Issue by johnallen24** [AAChartModel/AAChartKit-Swift#58]
I want to be able add a new data point every second to an existing AAChartView of type line chart. I am not sure how to do this. I know there is the aa_onlyRefreshTheChartDataWithChartModelSeries() function. The function takes the parameter chartModelSeries:Array<Any> but I am not sure exactly what a chartModelSeries is? Currently I am putting an updated array with the new data value appended at the end for this parameter. However, I get 

☠️☠️💀☠️☠️WARNING!!!!! THERE ARE SOME ERROR INFOMATION_______Error Domain=WKErrorDomain Code=4 "A JavaScript exception occurred" UserInfo={WKJavaScriptExceptionLineNumber=102, WKJavaScriptExceptionMessage=TypeError: undefined is not an object (evaluating 'chartSeries.setData'), WKJavaScriptExceptionColumnNumber=28, WKJavaScriptExceptionSourceURL=file:///Users/......AAChartView.html, NSLocalizedDescription=A JavaScript exception occurred}

Any help would be greatly appreciated.

**Comment by AAChartModel** [comment]
Maybe you need this https://www.highcharts.com/ios/demo/dynamic-update

**Comment by johnallen24** [comment]
Is that possible to create with AAChartKit?

**Comment by AAChartModel** [comment]
NO,It is come from Highcharts offical wrapper `Highcharts-iOS`,you can get it from here https://github.com/highcharts/highcharts-ios
